app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +12,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.AppCompat.NoActionBar">
+        android:theme="@style/Theme.AppCompat.NoActionBar"
+        tools:ignore="AllowBackup">
```

Comment:
is really necessary that you have the allowBackup in true?

Reply:
if not, please remove the ignore

Reply:
Refactored to "false", thanks Mau!